dyson_wiki.py

```diff
@@ -530,6 +530,7 @@ def print_wiki(data):
                          big hack. Omitted if empty.
     pre_item - Prerequisite items to research. Omitted if empty.
     is_hidden_tech - Is this normally hidden? Tends to go with pre_item.
+    is_obsolete - True when obsolete.
```

Comment:
This needs additional code format_tech to actually be included in the Lua table, but that can be added later if it's needed.